<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiveLogExportDialog</class>
 <widget class="QDialog" name="DiveLogExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export dive log files</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:subsurface-icon</normalon>
   </iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="General_tab">
      <attribute name="title">
       <string>General export</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="3" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="exportSelection">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string>Selection</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QRadioButton" name="exportSelected">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>Selected dives</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="exportAll">
               <property name="text">
                <string>All dives</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="title">
             <string>Options</string>
            </property>
            <widget class="QComboBox" name="CSVUnits_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>30</y>
               <width>102</width>
               <height>27</height>
              </rect>
             </property>
             <item>
              <property name="text">
               <string>Metric</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Imperial</string>
              </property>
             </item>
            </widget>
            <widget class="QCheckBox" name="anonymize">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>70</y>
               <width>111</width>
               <height>20</height>
              </rect>
             </property>
             <property name="text">
              <string>Anonymize</string>
             </property>
            </widget>
            <widget class="QCheckBox" name="diveinfo">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>90</y>
               <width>181</width>
               <height>23</height>
              </rect>
             </property>
             <property name="text">
              <string>Dive info in profle</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="exportFormat">
         <property name="title">
          <string>Export format</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="exportSubsurfaceXML">
            <property name="maximumSize">
             <size>
              <width>171</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Subsurface &amp;XML</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">exportGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="exportSubsurfaceSitesXML">
            <property name="text">
             <string>Subsurface dive sites XML</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">exportGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="exportUDDF">
            <property name="maximumSize">
             <size>
              <width>110</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>UDDF</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">exportGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="exportDivelogs">
            <property name="text">
             <string>di&amp;velogs.de</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">exportGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="exportDiveshare">
            <property name="text">
             <string>DiveShare</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">exportGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="exportCSVDetails">
            <property name="text">
             <string>CSV summary dive details</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">exportGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="exportCSV">
            <property name="text">
             <string>CSV dive computer dive profile</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">exportGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="exportProfileData">
            <property name="text">
             <string>CSV computed profile panel data</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">exportGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="exportProfile">
            <property name="text">
             <string>Dive profile</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">exportGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="exportWorldMap">
            <property name="text">
             <string>Worldmap</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">exportGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="exportTeX">
            <property name="text">
             <string>TeX</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">exportGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="exportLaTeX">
            <property name="text">
             <string>LaTeX</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">exportGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="exportImageDepths">
            <property name="text">
             <string>I&amp;mage depths</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">exportGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="description">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="HTML_tab">
      <attribute name="title">
       <string>HTML</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="GeneralOptions">
         <property name="title">
          <string>General settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QCheckBox" name="exportSubsurfaceNumber">
            <property name="text">
             <string>Subsurface numbers</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="exportSelectedDives">
            <property name="minimumSize">
             <size>
              <width>117</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Selected dives</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="exportStatistics">
            <property name="text">
             <string>Export yearly statistics</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QRadioButton" name="exportAllDives">
            <property name="minimumSize">
             <size>
              <width>117</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>All di&amp;ves</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="exportListOnly">
            <property name="text">
             <string>Export list only</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="exportPhotos">
            <property name="text">
             <string>Export photos</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="advanceOptions">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Style options</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="fontLabel">
            <property name="text">
             <string>Font</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="fontSelection"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="fontSizeLabel">
            <property name="text">
             <string>Font size</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="fontSizeSelection">
            <property name="currentIndex">
             <number>3</number>
            </property>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>12</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>14</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>16</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>18</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>20</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="themeLabel">
            <property name="text">
             <string>Theme</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="themeSelection">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string>Light</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sand</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DiveLogExportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DiveLogExportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exportCSV</sender>
   <signal>toggled(bool)</signal>
   <receiver>CSVUnits_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exportCSVDetails</sender>
   <signal>toggled(bool)</signal>
   <receiver>CSVUnits_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="exportGroup"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
